packages/EasyApiPlatform/.gitignore

```diff
@@ -1,4 +1,4 @@
 # Composer
 composer.lock
 /vendor
-/var
+**/var
```

Comment:
Why?

Reply:
To match directories anywhere in the repository.

Reply:
Double check with https://www.atlassian.com/git/tutorials/saving-changes/gitignore#git-ignore-patterns

Reply:
Yeah, I see, but do we have `/var` somewhere else?
Oh, is it for tests?

Reply:
yes, this is for tests.

Reply:
Got it, thanks.